certifier_service/certlib/certlib_proofs.go

```diff
@@ -1153,6 +1153,54 @@ func VerifySevAttestation(serialized []byte, k *certprotos.KeyMessage) []byte {
 		  byte dev_public_key[PUBLIC_KEY_SIZE];
 		};
  */
+
+/*
+ * TODO: Notes taken during call with Ye. To be deleted ...
+ *
+ * I need to probably do e2e implementation like this for CCA:
+ *
+ * Option 1: (John prefers this option)
+ *
+ *  - Figure out how to write this fn. How to verify evidence provided by CCA-shim?
+ *  - CCA-shim provides an attest function which generates an evidence report
+ *  - Need to write something to generate an evidence package on top of that evidence.
+ *  - Write a fn that calls CCA-Attest(), packages it together before sending it to
+ *    the Cert service.
+ *
+ *  - certify_me() will serialize evidence package and send it to Cert Service
+ *  - In this fn, VerifyKeystoneAttestation() seems to take serialized byte stream
+ *    and a key. The former seems to be serialiazed evidence.
+ *
+ *    Most straightforward way to verify is to have a public-key (may be 'k' here)
+ *    and verify it. In most cases, this key is called platform-cert. This is the cert
+ *    you get for the [say, SEV] machine. This cert can be used to verify evidence
+ *    generated by [this] machine.
+ *
+ *  - For shims, we probably have one public key-cert, if cert is not provided.
+ *
+ *    We can use SEV-implementation as a reference here
+ *
+ * Option 2: (Always available but we give up platform verification and trust the
+ *            SDK to do it.)
```

Comment:
Below is how SDK verify the attestaion report.

If you call islet_verify(), below steps are proceed.
1. Parse the attestation report to get `the realm(VM)'s signature`, `the public key` used at signing and `the realm payload` at [here](https://github.com/Samsung/islet/blob/main/sdk/src/verifier.rs#L11).
2. Verify them at [here](https://github.com/Samsung/islet/blob/main/sdk/src/verifier.rs#LL36C28-L36C34) according to  CBOR Object Signing and Encryption standard which is noted in [CCA spec section: A7.2.3](https://developer.arm.com/documentation/den0137/latest).

Reply:
This comment was for information and you've already done by calling `ccalib_Verify` :) Please resovle this comment. _(I don't have a permission to resolve comment.)_